plugin/pkg/scheduler/algorithm/priorities/priorities.go

```
@@ -294,15 +273,10 @@ func NewNodePreferAvoidPodsPriority(controllerLister algorithm.ControllerLister,
 	return nodePreferAvoid.CalculateNodePreferAvoidPodsPriority
 }
 
-func (npa *NodePreferAvoidPod) CalculateNodePreferAvoidPodsPriority(pod *api.Pod, nodeNameToInfo map[string]*schedulercache.NodeInfo, nodeLister algorithm.NodeLister) (schedulerapi.HostPriorityList, error) {
-	nodes, err := nodeLister.List()
-	if err != nil {
-		return nil, err
-	}
-
+func (npa *NodePreferAvoidPod) CalculateNodePreferAvoidPodsPriority(pod *api.Pod, nodeNameToInfo map[string]*schedulercache.NodeInfo, nodes []*api.Node) (schedulerapi.HostPriorityList, error) {
 	// TODO: Once we have ownerReference fully implemented, use it to find controller for the pod.
-	rcs, err := npa.controllerLister.GetPodControllers(pod)
-	rss, err := npa.replicaSetLister.GetPodReplicaSets(pod)
+	rcs, _ := npa.controllerLister.GetPodControllers(pod)
```

why ignore the error?